docker-compose.yml

```diff
@@ -21,3 +21,16 @@ services:
       - DB_NAME=test-db
       - DB_CONFIG_TABLE_NAME=test-1
       - MANIFEST=README.md
+
+  bapm-server:
+    image: bapm-server
+    network_mode: "host"
+    tty: true
+    build: ./bapm_server
+    environment:
+      - ES_HOST=localhost
+      - ES_PORT=9200
+      - BAPM_MQ_HOST=amqp://guest:guest@aw-sdx-monitor.renci.org:5672/%2F
```

Comment:
I'm afraid that we're leaking secrets here.  If we're only providing this as an example, that probably is fine. But there's a better way to manage secrets in docker-compose: 

https://pythonspeed.com/articles/build-secrets-docker-compose/

Reply:
Yes we'll use manifest later on, and move docker environ variables there. Also we'll add RabbitMQ security enhancement.

Relates to: https://github.com/atlanticwave-sdx/sdx-controller/issues/105